pkg/kube_events_manager/types/types.go

```diff
@@ -57,19 +58,12 @@ func (o ObjectAndFilterResult) Map() map[string]interface{} {
 		return m
 	}
 	// Add filterResult field only if it was requested
-	inJson := o.FilterResult
-	if inJson == "" {
-		m["filterResult"] = nil
-		return m
-	}
-	var res interface{}
-	err := json.Unmarshal([]byte(inJson), &res)
-	if err != nil {
-		log.Errorf("Possible bug!!! Cannot unmarshal jq filter '%s' result: %s", o.Metadata.JqFilter, err)
+	if o.FilterResult == nil {
```

Comment:
As I understand, this method is not used for go hooks, so FilterResult here can't be nil for jqFilter:

`res.FilterResult = []byte(filtered)`

https://github.com/flant/shell-operator/pull/267/files#diff-37508d544287271c578df624603d224bf1332acbb608e7c5dbfeea23e9ac72a0R70

Reply:
Fixed this. It is now Umarshalled from JSON, exactly as it was before my changes. If there was a `null` inside supposed JSON string, then it'll be nil.